src/Symfony/Component/Serializer/composer.json

```diff
@@ -33,7 +33,7 @@
         "symfony/http-foundation": "^4.4|^5.0|^6.0",
         "symfony/http-kernel": "^4.4|^5.0|^6.0",
         "symfony/mime": "^4.4|^5.0|^6.0",
-        "symfony/property-access": "^5.4|^6.0",
+        "symfony/property-access": "^5.4.4|^6.0.4",
```

Comment:
Can you elaborate on why these versions precisely? The next bugfix release for property-access (supposedly embedding this patch) will be v5.4.27 so I suppose this targets a specific change

Reply:
The reason is #45306. Without it the new test fails:

```
1) Symfony\Component\Serializer\Tests\SerializerTest::testCollectDenormalizationErrorsWithoutTypeExtractor
Failed asserting that TypeError Object (...) is an instance of class "Symfony\Component\Serializer\Exception\PartialDenormalizationException".
```

Reply:
I just noticed that 5.4.4 and 6.0.4 don't exist, so 5.4.5 and 6.0.5 are actually installed instead.